lib/twingly/url.rb

```diff
@@ -22,6 +22,8 @@ class URL
       Addressable::URI::InvalidURIError,
       PublicSuffix::DomainInvalid,
     ].freeze
+    DOT = "."
+    HYPEN = "-"
```

Comment:
should these be marked as private constants too? can't spot usage outside of this file anyway

Reply:
Yes :)